assets/survey_themes/fruity_twentythree/questiontypes/file_upload/file_upload.scss

```diff
@@ -1,3 +1,7 @@
 .file-preview fieldset span {
     margin-left: 0;
+}
+
+.uploadedfiles tbody tr td {
+    text-align: left;
```

Comment:
Needed ? Inherited value are not OK ?

Reply:
It would be center if inherited

Reply:
Sicne the text-aligne: left came from a specific package : must be fixed in this specific package.

Else : it was centered in 5.X : it's an issue ?

Reply:
5x also has align left for file upload table.
I'd love to use bootstrap classes, but the way RTL is handled in the new theme vs the old themes, makes it impossible to use those classes.

Reply:
Yes, but not with skelvanilla.

Reply:
Still : since text-align is set in https://github.com/LimeSurvey/LimeSurvey/blob/3ee3a558a71a34da2be44eb4976b05b5bd924acd/assets/packages/questions/upload/styles/uploader.css#L91 rtl system must be set in https://github.com/LimeSurvey/LimeSurvey/blob/3ee3a558a71a34da2be44eb4976b05b5bd924acd/assets/packages/questions/upload/styles/uploader.css#L91